TestAssets/TestProjects/TestAppWithContentPackage/project.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "NETStandard.Library": "1.0.0-rc2-23704",
+    "System.Runtime": "4.0.21-rc2-23704",
```

Comment:
Few questions:
- Why is Stdlib a different version to everything else?
- Why are you pulling in System.Runtime and System.Console, those are part of stdlib.